pkg/apis/extensions/latest/latest.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
```

Comment:
There should be a single 'latest' package. There should *not* be a latest-package-per-group.

'Latest' is poorly and confusingly named, for which I apologize. It functions as a central place for all groups to register themselves, and all group-consumers to go to find their group. Making a different latest for every group defeats the purpose.